packages/playground/src/components/Input.vue

```diff
@@ -20,6 +20,16 @@ const props = defineProps<{
   model: EditorJSModel;
 }>();
 
+// three types of inputs:
+// ?type=contenteditable
+// ?type=input
+// ?type=textarea
+let type = (new URL(window.location.href).searchParams.get('type') ?? 'contenteditable') as 'contenteditable' | 'input' | 'textarea';
```

Comment:
you can use vue component props instead

Reply:
I think it will be simpler if I read it directly.

Reply:
I still think that props are more suitable here. Query params are related to the page itself, when props are related to the component instance. Maybe we'll need several Inputs of different types on the Playground.

Props will be quite simpler:

**Input.vue**

```ts
defineProps<{
  type: 'contenteditable' | 'input' | 'textarea'
}>()
```

**Playground.vue**

```html
<Input type="contenteditable" />
<Input type="input" />
```

Reply:
For now, we don't support multiple input elements on the same page, and cannot auto-unmount input elements.